File: DailyChallenge/JavaScript/20260106_vOwElcAsE.js
function vowelCase(str) {
  const regExVowel = /[aeiou]/gi;
  const regExAlphabet = /[A-Za-z]/gi;
  let newString = '';

  for(let i = 0; i < str.length; i++) {
    let char = str[i];

    if(char.match(regExVowel)) {
      char = char.toUpperCase();
    }
    else if(char.match(regExAlphabet)) {
      char = char.toLowerCase();
    }

    newString += char;
  }

  return newString;
}

File: Coding Challenges/DailyChallenge/Python/20260106_vOwElcAsE.py
import re

def vowel_case(s):
    regExVowel = '[aeiou]'
    regExAlphabet = '[A-Za-z]'

    newString = ''

    for char in s:
        if re.match(regExVowel, char, re.IGNORECASE):
            newString += char.upper()
        elif re.match(regExAlphabet, char, re.IGNORECASE):
            newString += char.lower()
        else:
            newString += char

    return newString